Sources/CombineBluetooth/Models/CentralManager.swift

```diff
@@ -14,7 +23,17 @@ public protocol CentralManager: BluetoothManager {
     func connect(_ peripheral: BluetoothPeripheral) -> AnyPublisher<BluetoothPeripheral, BluetoothError>
     func connect(_ peripheral: BluetoothPeripheral, options: [String : Any]) -> AnyPublisher<BluetoothPeripheral, BluetoothError>
     func peripheral(for uuid: UUID) -> BluetoothPeripheral?
+    
     // TODO: Nice to have, but complicate to handle single delegate (subscribing again with different options should end prior observations, which can
     //       be a bit unexpected).
     // - (void)registerForConnectionEventsWithOptions:(nullable NSDictionary<CBConnectionEventMatchingOption, id> *)options NS_AVAILABLE_IOS(13_0);
 }
+
+public struct CentralManagerFactory {
```

Comment:
nit: this is not Java :) (of course I have no better idea for a name. Naturally I would put this static method into the type that is created, but with generics and protocols there is no obvious place)

Reply:
True, also tried to put this in a `static func` within in the context of `CentralManager` but it seemed not doable for the type resolving, at least I didn't find a way within a reasonable time.
Didn't came up with a better name everybody would vaguely understand than that factory, but I'd always be open for suggestions. I'd rather have the bit of Java flair than some obfuscated stuff people would have a hard time getting what it does

Reply:
If I am not missing a point, we can also get rid of this. 
CoreBluetoothCentralManager(centralManager: CBCentralManager) exactly does the same.